Automattic/wp-calypso
# Jetpack New Pricing page - Disable manage subscription button for user who isn't a plan purchaser

Completes 1202796695664022-as-1202883205133640/f

#### Changes proposed in this Pull Request

* Create a hook `useItemInfo` to get item specific details like `isNotPlanOwner`
* Use the hook and `isOwned` to disable or enable *Manage Subscription* button 
* Reuse the *OwnerInfo* component to show tooltip for disabled button
* Fix styling for the tooltip icon

#### Testing instructions

- Create a JN site and connect Jetpack
- Add a secondary admin to the site and login to the new admin account.
- Do any one of these
    * Click on Jetpack Cloud live link below and add this to the end of URL `?flags=jetpack/pricing-page-rework-v1`
    * or boot up this PR 
        * Run `git fetch && git checkout add/jp-pricing-rework/fix-wpcom-styles`
        * Run `yarn start`
        * Goto `http://calypso.localhost:3000/plans/?flags=jetpack/pricing-page-rework-v1`
- Confirm that you see the new pricing UI
- Confirm that the already purchased product has *Manage Subscription* button and it is disabled 
- Confirm that there is info icon next to *Manage Subscription* and it shows tooltip
- Now login back to main account who purchased the subscription and confirm that *Manage Subscription* is not disabled

**Screenshots**
#### A different user who did not purchase the subscription

![Screenshot 2022-09-03 at 2 05 17 AM](https://user-images.githubusercontent.com/2027003/188231883-6f95c5c9-0aef-4c93-9021-d219ad4db74e.png)

#### User who purchased the subscription 

![Screenshot 2022-09-03 at 2 01 27 AM](https://user-images.githubusercontent.com/2027003/188231336-5afeab52-668f-4a16-a631-f52e345b11b4.png)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202883205133640
